fiat-roles/src/test/groovy/com/netflix/spinnaker/fiat/providers/DefaultServiceAccountProviderSpec.groovy

```diff
@@ -44,24 +48,26 @@ class DefaultServiceAccountProviderSpec extends Specification {
   @Unroll
   def "should return all accounts the specified groups has access to"() {
     when:
-    def result = provider.getAllRestricted(input.collect { new Role(it) } as Set)
+    def result = provider.getAllRestricted(input.collect { new Role(it) } as Set, fiatAdmin)
 
     then:
     CollectionUtils.disjunction(result, expected).isEmpty()
 
     when:
-    provider.getAllRestricted(null)
+    provider.getAllRestricted(null, FiatAdminRole.NONE)
 
     then:
     thrown IllegalArgumentException
 
     where:
-    input           || expected
-    []              || []
-    ["a"]           || [aAcct]
-    ["b"]           || []
-    ["c"]           || []
-    ["a", "b"]      || [aAcct, bAcct]
-    ["a", "b", "c"] || [aAcct, bAcct]
+    input           || fiatAdmin           || expected
```

Comment:
In Spock test tables, you generally put a single pipe `|` between each input/output, and the double pipe `||` between to signal the difference between inputs and outputs.

For example:
```
input1 | input2 | input3 || output1 | output2
```